automation/Dockerfile

```diff
@@ -19,8 +19,6 @@ RUN apt-get clean && rm -rf /var/lib/apt/lists/partial \
   /autobase/automation/requirements.txt \
   && ansible-galaxy install --force -r \
   /autobase/automation/requirements.yml \
-  && ansible-galaxy install --force -r \
-  /autobase/automation/roles/consul/requirements.yml \
```

Comment:
Why did you delete it? This is necessary to support a scheme using Consul as a DCS.

Reply:
Only `ansible.utils` was in there which is already in the collections global `requirements.yml`. All requirements of the collection get installed automatically when installing the collection via `ansible-galaxy`.